service/src/main/java/org/cbioportal/service/util/AppConfig.java

```diff
@@ -0,0 +1,28 @@
+package org.cbioportal.service.util;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableAsync;
+import org.springframework.scheduling.annotation.AsyncConfigurer;
+import org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor;
+
+import java.util.concurrent.Executor;
+
+@Configuration
+@EnableAsync
+public class AppConfig implements AsyncConfigurer {
```

Comment:
Looks good to me. I'm going to add Ben as a reviewer because of this though. I think it's correct, but he should know when we make a significant addition to the backend config.